src/main/kotlin/com/sourcegraph/cody/edit/sessions/FixupSession.kt

```diff
@@ -411,6 +399,8 @@ abstract class FixupSession(
     if (project.isDisposed) return
     performedActions.forEach { it.dispose() }
     editor.document.removeDocumentListener(documentListener)
```

Comment:
@pkukielka, the problem was that we called it twice - we should not see more errors from this one

Reply:
Great :)